### ISSUE TITLE
REMatch Kernel overflow

### ISSUE THREAD

**Issue by Abdelrahmansameh** [SINGROUP/dscribe#25]
Hello, I'm wondering if the REMatch kernel works or not, as I seem to be having the same error when I try to use it.
Here is an example code for reference 

water = build.molecule("H2O")`
s =  descriptors.SOAP(species=["H", "O", "C"], rcut=2, nmax=5, lmax=5, crossover=True)
t = s.create(water)
k = kernels.REMatchKernel(metric = "linear")
print(k.get_global_similarity(k.create([t, t])))

This snippet raises an overflow error when I run it. In fact I can't seem to get it to work with any molecule. 

Thank you,

**Comment by lauri-codes** [comment]
Hi @Abdelrahmansameh,

With the REMatch kernel, you will have to be careful about large numbers ending up in the Sinkhorn algorithm. Depending on your chosen metric, you may need to for example normalize the input in some way. This is also mentioned in the [REMatch tutorial](https://singroup.github.io/dscribe/tutorials/kernels.html).

With the SOAP descriptor and a linear metric in particular, just normalize the input vectors to unit norm, so add this step between the descriptor creation and the kernel calculation:
```
from sklearn.preprocessing import normalize

t = normalize(t)
```

This normalization step is not done by default by DScribe, because the need and best way to to it really depends on your kernel. E.g. with a Gaussian metric, this normalization is not necessary.

**Comment by Abdelrahmansameh** [comment]
Thank you @lauri-codes for the quick response! I will go ahead and do that.
